### PR TITLE
citation: 18-test deep-dive — 16p/1pa/0f/1ab (round 15)

### LINKED ISSUE #276: citation skill deep-dive
see eval/briefs/citation.md

### PR DESCRIPTION
**Pull request by T-FEH**
Closes #276

## Summary

- Builds a comprehensive 18-test corpus covering all citation source types and edge cases
- Iterative eval/annotate/fix cycle across 15 rounds, ending at **16 pass / 1 partial / 0 fail / 1 abort**
- 0 fails achieved for the first time; the single abort (ut_citation_010) is a recurring API wall-clock timeout, not a skill regression

## What changed in SKILL.md

**Routing stability (012):** Moved ROUTING section to absolute first position before narration and file reads, with explicit trigger phrases ("I found", "I just found", "Add it as a source") and a guard for combined add+format requests.

**Write-first rule for missing locators (017):** Added instruction anchored in Step 2 where the skill processes each field — notes explicitly confirm the value is missing → write [LOCATOR NOT RECORDED] immediately, do NOT pause to ask the user.

**Negative-search scope fabrication (007):** Replaced template example with neutral data (no Flynn/Schuylkill names) and added a query-field scope rule: the query field is the authoritative search scope; notes saying "no results in [COUNTY]" are geographic observations, not evidence of a county-scoped search.

**Probate where_within format (006):** Added a note that where_within holds only the physical locator (Will Book vol/page or the unknown-marker) — never the document title or party name.

**Derivative index format (004):** Combined physical locators + [HEAD OF HOUSEHOLD] entry format; explicit rule that "digital index" not "digital image" for Ancestry-style derivative citations.

## Run log

eval/runlogs/unit/citation/v1_2026-06-14_21-34-48.json — 18 tests, round 15

## Remaining items

| Test | Dim | Score | Note |
|------|-----|-------|------|
| 002 | EE | 2 | Judge critiques framing of where_within analysis; all 7 other dims pass at 3 — annotation pending |
| 010 | — | abort | Recurring API wall-clock timeout; judge skipped, 0 dims recorded |

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: eval/tests/unit/citation/url-only-expansion.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "test": {
+    "id": "ut_citation_008",
+    "skill": "citation",
+    "name": "Expand a bare URL into a full EE citation",
+    "type": "positive",
+    "description": "User pastes a raw FamilySearch URL with a long query string and asks the skill to turn it into a proper citation. Tests: (1) the skill treats the URL as a starting point only, (2) strips everything after the first '?' from the URL, (3) expands it into all six citation_detail fields, and (4) produces a formatted EE citation string. No scenario source is needed — the URL is provided inline in the user message.",
```
Comment (Leduthet) [inline eval/tests/unit/citation/url-only-expansion.json]: "the URL is provided inline in the" just a typo. Should be online not inline.

**Review by Leduthet — CHANGES_REQUESTED** [review]
There is a small typo. Thanks!

**Review by Leduthet — APPROVED** [review]
